Source/ProgressionSystemRuntime/Public/Data/PSWorldSubsystem.h

```diff
@@ -0,0 +1,104 @@
+// Copyright (c) Valerii Rotermel & Yevhenii Selivanov
+
+#pragma once
+
+#include "PSTypes.h"
+#include "Subsystems/WorldSubsystem.h"
+#include "PSWorldSubsystem.generated.h"
+
+
+/**
+ * Implements the world subsystem to access different components in the module 
+ */
+UCLASS(BlueprintType, Blueprintable, Config = "ProgressionSystem", DefaultConfig)
+class PROGRESSIONSYSTEMRUNTIME_API UPSWorldSubsystem : public UWorldSubsystem
+{
+	GENERATED_BODY()
+
+	DECLARE_DYNAMIC_MULTICAST_DELEGATE_OneParam(FCurrentRowDataChanged, const FPSRowData&, SavedProgressionRowDataInternal);
```

Comment:
1. This macro is under 'private' access specifier which will could fail the build or give us compile error on different platforms. Please move it under `public`, right above OnCurrentRowDataChanged.
2. You don't need 'Internal' in the same argument name. It's not protected property, but just argument.

So, so it will look next:
```
// ... here all previous code in 'public' access like static UPSWorldSubsystem& Get();

DECLARE_DYNAMIC_MULTICAST_DELEGATE_OneParam(FCurrentRowDataChanged, const FPSRowData&, SavedProgressionRowData);

/* Delegate for informing row data changed */
UPROPERTY(BlueprintAssignable, Category = "C++")
FCurrentRowDataChanged OnCurrentRowDataChanged;
```

Reply:
got it, silly mistake. done.